unicorn_contracts/pyproject.toml

```diff
@@ -7,9 +7,9 @@ exclude = ["src/events/"]
 packages = [{ include = "contracts_service", from = "src" }]
 
 [tool.poetry.dependencies]
-python = "^3.9"
-boto3 = "^1.24.12"
-aws-lambda-powertools = "^1.27.0"
+`python = "^3.10"
```

Comment:
Typo here, need to remove that leading '`'

Reply:
Ooops. Will fix

Reply:
Fixed